src/swsssdk/scripts/sonic-db-cli

```diff
@@ -20,4 +21,10 @@ else:
         print >> sys.stderr, msg
     else:
         client = dbconn.get_redis_client(dbname)
-        print client.execute_command(" ".join(sys.argv[2:]))
+        if op == 'EVAL':
+            lua_script = sys.argv[3]
+            numkeys = sys.argv[4]
+            keys_args = [] if numkeys == 0 else sys.argv[5:]
+            print client.execute_command(op, lua_script, numkeys, *keys_args)
+        else:
+            print client.execute_command(" ".join(sys.argv[2:]))
```

Comment:
Can you unify both branches? `execute_command` is quite general for both cases.

Reply:
DONE